blockbase/block-templates/index.html

```diff
@@ -3,7 +3,7 @@
 <!-- wp:query {"tagName":"main","layout":{"inherit":true}} -->
 <main class="wp-block-query">
 <!-- wp:post-template -->
-<!-- wp:group -->
+<!-- wp:group {"layout":{"inherit":true}} -->
```

Comment:
this is already inside a layout inherit, even though this solves the bug, the original markup should have been working in any case